superset/views/api.py

```diff
@@ -18,7 +18,7 @@ class Api(BaseSupersetView):
     @api
     @handle_api_exception
     @has_access_api
-    @expose('/v1/query', methods=['POST'])
+    @expose('/v1/query/', methods=['POST'])
```

Comment:
hmm, I thought we decided to drop the `/` to avoid calling the endpoint with `/v1/query/?slice_id...`, no?